data/_models/shared/Resources.csv

```diff
@@ -68,12 +70,12 @@ Resources,,contributors,heading,,,,,,,,,List of relevant collaborations,"DataCat
 Resources,,people involved,refback,,,,Contacts,,resource,,,Persons involved and their role(s),"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE"
 Resources,,contact point,ref,,,,Contacts,,,,dcat:contactPoint,This attribute is written during migration of data from staging areas and used to produce valid dcat output. Contact point for this resource,DataCatalogueFlat
 Resources,,organisations involved,refback,,,,Organisations,,resource,,,Organisations involved and their role(s),"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,UMCGCohortsStaging,NetworksStaging,UMCUCohorts,INTEGRATE"
-Resources,,publisher,ref,,,,Organisations,,,,dcterms:publisher,This attribute is written during migration of data from staging areas and used to produce valid dcat output. Organisation that published this resource,DataCatalogueFlat
+Resources,,publisher,ref_array,,,,Organisations,,,,dcterms:publisher,This attribute is written during migration of data from staging areas and used to produce valid dcat output. Organisation that published this resource,DataCatalogueFlat
```

Comment:
are you sure this is allowed? makes sense otherwise.

Reply:
I'm not entirely sure. The Catalogues table also has the column publisher and it was different than the record in Resources. In order to combine the tables, this needed to be an array.

Reply:
@BrendaHijmans double check what HRI core says about cardinality

Reply:
maybe undo for now and move to later PR